qa/tasks/rgw.py

```diff
@@ -13,6 +13,7 @@
 from teuthology import misc as teuthology
 from teuthology import contextutil
 from teuthology.orchestra.run import CommandFailedError
+from teuthology.exceptions import ConfigError
```

Comment:
could move this line 1 line up. less inversion count this way.

Reply:
Actually CommandFailedError is not even used. I might as well remove the line.